configs/raspberrypi_common_config

```diff
@@ -0,0 +1,90 @@
+# Raspberry Pi does not support GRUB bootloader integration, fallback to U-boot.
+MENDER_GRUB_EFI_INTEGRATION=n
+
+# Nothing to copy
+MENDER_COPY_BOOT_GAP=n
+
+# 4MB alignment
+MENDER_PARTITION_ALIGNMENT="4194304"
+
+# compress the images to save some space
+MENDER_COMPRESS_DISK_IMAGE="gzip"
+
+# from the "bookworm" release onwards, the Raspberry Pi OS
+# specific modifications expect the boot partition to be mounted
+# at /boot/firmware
+MENDER_BOOT_PART_MOUNTPOINT="/boot/firmware"
```

Comment:
@TheYoctoJester This should be:
```suggestion
MENDER_BOOT_PART_MOUNT_LOCATION="/boot/firmware"
```

And, additionally, we need to set it in `/configs/mender_convert_config` to avoid the unbound variable error. Something like:
```
# User specified boot mount location. If left blank it uses `/boot/efi
# when `MENDER_GRUB_EFI_INTEGRATION` is set, otherwise `/uboot`.
MENDER_BOOT_PART_MOUNT_LOCATION=""
```

Reply:
Has this been addressed?